mapper/pchain/tx_dependency.go

```diff
@@ -167,6 +177,16 @@ func (d *SingleTxDependency) GetUtxos() map[avax.UTXOID]*avax.UTXO {
 			outsToAdd = append(outsToAdd, unsignedTx.Stake()...)
 		case *txs.TransferSubnetOwnershipTx:
 			outsToAdd = append(outsToAdd, unsignedTx.Outputs()...)
+		case *txs.ConvertSubnetToL1Tx:
```

Comment:
Is it easy/worth to add unit tests for these cases?

Reply:
We are doing the same operations for all the tx types which is to add outputs to an array. I don't think testing the new transaction types will make any difference here.